pom.xml

```diff
@@ -9,7 +9,8 @@
     </parent>
 
     <artifactId>jsch</artifactId>
-    <version>${revision}${changelist}</version>
+    <!-- TODO: Switch separator from '.' to '-' next time jsch dependency is updated. -->
+    <version>${revision}.${changelist}</version>
```

Comment:
The current plugin version is 0.1.55.2. 0.1.55 is the version of jsch, and the `.2` suffix is for this plugin. Normally we use `-` to separate the library version from the plugin version for API plugins like this, but I do not know for sure whether `0.1.55.2` would be considered older than `0.1.55-60.vfa_d521b_a_1eb_e` and so I just kept using `.` for now. If `0.1.56` ever gets released we could safely switch to `-` at that time.

If anyone knows for sure that Maven and Jenkins and anything else we might care about would consider `0.1.55-60.vfa_d521b_a_1eb_e` newer than `0.1.55.2`, then we could switch right now.

Reply:
don't hold your breath.
https://sourceforge.net/p/jsch/mailman/jsch-users/thread/22608f9c-3308-ef6a-0849-0166194f0840%40mlink.com/#msg37027593

I would suggest migrating anything that requires this to something else instead like mina ssh

Reply:
Yes I know it is essentially abandoned, I am just noting the version number discrepancy here. 

> I would suggest migrating anything that requires this to something else instead like mina ssh

Sure I agree, but I do not have time to work on that. I am just trying to get a release out for #17 to simplify BOM testing.